cfme/tests/infrastructure/test_host.py

```diff
@@ -87,6 +92,6 @@ def test_multiple_host_bad_creds(setup_provider, provider):
                            'validate_host': cfme_host["name"]})
 
     sel.click(credential_form.validate_multi_host)
-    flash.assert_message_match('Cannot complete login due to an incorrect user name or password.')
+    flash.assert_message_match(msgs[provider.type])
```

Comment:
REQUIRED: @istein1, we are trying to get away from using provider.type and et cetra. 
Please replace this with provider.one_of() construction.